nav_msgs/OccupancyGrid grid1
nav_msgs/OccupancyGrid grid2